ex1/tema1.js

```diff
@@ -0,0 +1,7 @@
+
+function squareIt(number) {
```

Comment:
```suggestion
function getSquareOf(number) {
```

(optional)Desi imi place cum suna numele :)  Este totusi o practica buna sa inceapa functiile cu prefixul "get" care inseamna ca returneaza ceva